actors/builtin/paych/paych_actor.go

```diff
@@ -93,6 +93,8 @@ type UpdateChannelStateParams struct {
 // A voucher is sent by `From` to `To` off-chain in order to enable
 // `To` to redeem payments on-chain in the future
 type SignedVoucher struct {
+	// PaymentChannelAddr is the address of the payment channel this signed voucher is valid for
+	PaymentChannelAddr addr.Address
```

Comment:
Let's just call it `ChannelAddr`, I think "payment" is sufficiently implied (remember to regenerate cbor).

Reply:
This is done.

Reply:
This is done.